src/components/UserSelector.tsx

```diff
@@ -38,10 +29,23 @@ export const UserSelector: React.FC<Props> = ({
     return () => {
       document.removeEventListener('click', handleDocumentClick);
     };
-    // we don't want to listening for outside clicks
-    // when the Dopdown is closed
   }, [expanded]);
 
+  const showUsers = users.map(user => (
+    <a
+      key={user.id}
+      href={`#user-${user.id}`}
+      onClick={() => {
+        dispatch(setAuthor(user));
+      }}
+      className={classNames('dropdown-item', {
+        'is-active': user.id === selectedUser?.id,
+      })}
+    >
+      {user.name}
+    </a>
+  ));
```

Comment:
This code block correctly maps over the users array to create a list of user links. Ensure that the users array is not empty before mapping to avoid potential errors.